osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -569,6 +615,22 @@ protected override void LogoExiting(OsuLogo logo)
             logo.FadeOut(120, Easing.Out);
         }
 
+        private void updateWedgeVisibility()
```

Comment:
Side note but there's a worst case scenario here when switching to a ruleset with converts disabled and simultaneously not having any non-convert maps imported:

![Screenshot 2025-06-04 at 12 01 05](https://github.com/user-attachments/assets/fdb2455c-051c-4ff8-9e51-8b15dae29e65)

Tested with a custom ruleset here but this could technically happen with any of the defaults too.

The left side hiding is fine but I dunno about the filter control hiding. Seems weird.

Reply:
seems bad, will adjust